tasks/the-build.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0"?>
+
+<!--
+  @file the-build.xml
+  Contains the-build's environment-specific property loading.
+
+  This file is included in the build.dist.xml template with:
+    <import file="vendor/palantirnet/the-build/tasks/the-build.xml" />
+
+  Copyright 2016, 2017, 2018 Palantir.net, Inc.
+  -->
+
+<project name="the-build" default="status">
+
+    <!-- Provide the <includeresource /> task. -->
+    <taskdef name="includeresource" classname="TheBuild\IncludeResourceTask" />
+
+
+    <!-- Alias the build directory to the Phing start directory. -->
+    <property name="build.dir" value="${application.startdir}" />
+
+    <!-- Use the project directory name as the project name -->
+    <basename property="projectname" file="${build.dir}" suffix="local" />
+
+    <!-- Provide the path to the-build's code for loading templates and defaults -->
+    <!-- Phing maps the project name "the-build" to "the_build". Should we use the_build instead to avoid this confusion? -->
```

Comment:
Sticking with `the-build` seems good to me.  It's pretty common for the dash/underscore mapping to happen, and sticking with the dash seems like it may be better for URLs and such.